### DISCUSSION TITLE
How to get access token and id from accounts document?

### DISCUSSION THREAD

**Discussion by sk0le** [nextauthjs/next-auth#1679] [Help]
**Your question**

How to get access token and id from accounts document - https://prnt.sc/1186oi6

**What are you trying to do**

I'm trying to access to "accesToken" and "providerAccountId"

**Reproduction**

Just basic setup with mongodb and typescript

**Feedback**

https://next-auth.js.org/configuration/callbacks - found this for callbacks but it feels incomplete

**Comment by balazsorban44** [comment]
What do you feel was incomplete in the docs?

Using the jwt and session callbacks, you can persist any information in your session.

account is the 3rd argument in the jwt callback, and is exposed on first login. what you return in jwt will be available in the session callback's second argument. what you return in session will be exposed to the client, so you can filter out things that are only necessary when reading session server-side for example.
